pkgs/by-name/ne/netscanner/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, makeWrapper
+, iw
+,
+}:
+let
+  pname = "netscanner";
+  version = "0.4.0";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  src = fetchFromGitHub {
+    owner = "Chleba";
+    repo = "netscanner";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

Avoids branch/tag confusion, helpful for updates made by bots.